drivers/quark/driver.go

```diff
@@ -203,7 +203,7 @@ func (d *Quark) Put(ctx context.Context, dstDir model.Obj, stream model.FileStre
 		}
 		md5s = append(md5s, m)
 		partNumber++
-		up(100 - int(left/sizeDivide100))
+		up(100 * (total - left) / total)
```

Comment:
Here is always 0 or 100 as the result of `int64`/`int64` is also int64, In my humble opinion